lib/background/main_background.dart

```diff
@@ -75,6 +75,8 @@ class BackgroundReceiver implements TimelineCallbacks {
   }
 
   void onWatchConnected(PebbleDevice watch) async {
+    await (await preferences).reload();
```

Comment:
Can you cache the result of `await preferences` inside this function? If you can, it might make for more readable code.